lib/Swagger/src/Discussion/Api/SitesApi.php

```diff
@@ -211,10 +196,6 @@ public function createSiteWithHttpInfo($site, $x_wikia_schwartz = null)
                     $data = $this->apiClient->getSerializer()->deserialize($e->getResponseBody(), '\Swagger\Client\Discussion\Models\HalProblem', $e->getResponseHeaders());
                     $e->setResponseObject($data);
                     break;
-                case 401:
-                    $data = $this->apiClient->getSerializer()->deserialize($e->getResponseBody(), '\Swagger\Client\Discussion\Models\HalProblem', $e->getResponseHeaders());
-                    $e->setResponseObject($data);
-                    break;
                 case 403:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This case's code block is the same as the block for the case on line 195. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)